scan.js

```diff
@@ -526,14 +534,24 @@ async function main () {
   // Start the browser
   printColorful('green', '[+] Starting browser...')
   const options = { headless: argv.headless ? 'new' : false }
+
   if (argv.proxy) {
-    printColorful('green', `[+] Setting proxy to ${argv.proxy}...`)
-    options.args = []
-    options.args.push(`--proxy-server=${argv.proxy}`)
-    printColorful('green', '  [+] Disabling Certificate Validation...')
-    options.args.push('--ignore-certificate-errors')
+      printColorful('green', `[+] Setting proxy to ${argv.proxy}...`)
+      options.args = options.args || [];  // Ensure args is initialized
+      options.args.push(`--proxy-server=${argv.proxy}`)
+      printColorful('green', '  [+] Disabling Certificate Validation...')
+      options.args.push('--ignore-certificate-errors')
+  }
+ //Add No-Sandbox option
+  if (argv.nosandbox) {
+      options.args = options.args || [];  // Ensure args is initialized
+      options.args.push('--no-sandbox');
+      printColorful('green', '  [+] Launching without sandbox...');
   }
+
   const browser = await pt.launch(options)
+  
+  //const browser = await pt.launch(options)
```

Comment:
Hi there!
First of all: Thank you very much for contributing to this tool! :)

The above line looks like left-over development code. Could you please remove that line so that I could accept your PR?

Thanks again and have a great day,

Lauritz